pkg/universe.dagger.io/x/vgjm456@qq.com/helm/helm.cue

```diff
@@ -0,0 +1,42 @@
+package helm
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/docker"
+)
+
+#Install: {
+	releaseName: string
+	chartName:   string
+	repoName:    string
+	repository:  string
+	kubeconfig:  dagger.#Secret
+	version:     string | *"latest"
+	commands: [
+		{
+			name: "repo"
+			args: ["add", "\(repoName)", "\(repository)"]
+		},
+		{
+			name: "install"
+			args: ["\(releaseName)", "\(repoName)/\(chartName)"]
+		},
+	]
+	docker.#Build & {
+		steps: [
+			docker.#Pull & {
+				source: "index.docker.io/alpine/helm:\(version)"
+			},
+			for cmd in commands {
```

Comment:
That's a great idea! I think it would be more readable if you directly write those 2 `docker.#Run`
You could also just run a multi-line script